app/uk/gov/hmrc/traderservices/views/CreateCaseConfirmationView.scala.html

```diff
@@ -86,14 +86,24 @@ <h1 class="govuk-panel__title">
         messageKey = "view.create-case-confirmation.feedback"
       )
     }
-
-    @html.h2("view.create-case-confirmation.print-receipts.heading")
-
+    <div id="confirmation-heading">
+      @html.h2("view.create-case-confirmation.print-receipts.heading")
+     </div>
+     <div >
+       <noscript>
+         @html.h2("view.amend-case-confirmation.info.no-javascript.heading")
```

Comment:
Incredibly picky and not even necessary but you could use the create-case text here for consistency in the naming OR just use one message for both seeing as it's the same? Honestly sorry for even mentioning it seems super picky cuz it's just _sliiiightly_ inconsistent - it works perfectly!